File: CodeParser.Test/Examples/HelloWorld.java
package demo;

import java.io.File;
import java.util.List;

public interface ISpeech {  
   final int count=0;
   void sayHello(String word);
}

public class JavaHelloWorld extends ISpeech {

	public String _name;
	
	public String getName() {
		return this._name;
	}
	
	public void setName(String name) {
		this._name=name;
	}
	
	public void sayHello(String word) {
		System.out.println(word);
	}
}

File: CodeParser.Test/Examples/HelloWorld.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.IO;

namespace Demo
{
    public interface ISpeech
    {
        string Name { get; }
        void SayHello(string word);
    }

    public delegate void SayHelloHandler(string word);

    public class CSharpHelloWord : ISpeech
    {
        public event SayHelloHandler OnSayHello;

        public string owner;
        public string Owner
        {
            get { return this.owner; }
            set { this.owner = value; }
        }

        public void SayHello(string word)
        {
            Console.WriteLine(word);
        }
    }

    public enum SayMode
    {
        Default,
        Custom
    }
}
